lerobot/scripts/train.py

```diff
@@ -425,7 +425,8 @@ def evaluate_and_checkpoint_if_needed(step, is_online):
         dataloading_s = time.perf_counter() - start_time
 
         for key in batch:
-            batch[key] = batch[key].to(device, non_blocking=True)
+            if key != 'language_instruction':
+                batch[key] = batch[key].to(device, non_blocking=True)
```

Comment:
This is probably a separate change, but how should we handle language instructions? We could either use a tokeniser or the languagew model itself here to ensure the data is placed into the device (GPU) from the start, or do it in the model though that may be less performant as not all instructions run in CUDA?